README.md

```diff
@@ -51,11 +49,11 @@ Potete suggerire cambiamenti in vari modi:
 
 + chi fosse invece più familiare con GitHub, può spingersi ad utilizzare il flusso di lavoro tipico in GitHub per modifiche sostanziali, che sfocia tipicamente in una ***pull request***, ma non è essenziale e non lo esigiamo in nessun modo, gli *issues* vanno benissimo
 
-+ al limite, va bene anche **una semplice segnalazione via mail** (in effetti, misura *anti-spam*, l'indirizzo è accessibile in *GitHub* solo *dopo* aver effettuato il login), sapendo che inviare mail espone a *metadata recollection*, ma *noi* stiamo comunque dalla parte della legalità e della Costituzione (potrete comunque usare *GPG* per segnalarci punti deboli del manifesto)
++ al limite, va bene anche **una semplice segnalazione via mail** (in effetti, misura *anti-spam*, l'indirizzo è accessibile in *GitHub* solo *dopo* aver effettuato il login), sapendo che inviare mail espone a *metadata recollection*, ma *noi* stiamo comunque dalla parte della legalità e della Costituzione. Potrete comunque usare *GPG* per segnalarci punti deboli del manifesto
 
 + potete scrivere **un commento alla** [**presentazione in Diaspora**](https://mondiaspora.net/posts/ab05452057880138212d0e36a79e7653)
```

Comment:
((( Questa non l'hai toccata ma ne approfitto per commentare: ci rendiamo conto che chiedere di commentare in Diaspora TANTO per dare l'approvazione COME per suggerire modifiche non va bene. Mi sa che questa modalità di suggerire modifiche la dovremo eliminare. Che te ne pare? Se poi ci fossero suggerimenti in Diaspora, pazienza si gestiscono comunque, ma il luogo ideale è evidentemente GitHub. )))

Reply:
((( OK su questo mi hai già rispoto in altro canale. )))